stdlib/public/TensorFlow/Gradients.swift

```diff
@@ -25,19 +25,16 @@
 // Each function in this file is the VJP of some corresponding function
 // defined in Ops.swift with respect to all arguments. The attribute
 // '@differentiable(reverse, vjp: ...)' is used to define the VJP for a
```

Comment:
Update the comment here because “reverse” in gone from @differentiable.

Reply:
Thanks! Done in https://github.com/apple/swift/pull/21916/commits/da4c2aef450c76f6d1fe14550d1d7e0d9d751fe9.